packer/ansible/playbook.yml

```diff
@@ -95,10 +99,14 @@
     - ncats_webd
     - ncats_webui
     - docker
+  vars:
+    maxmind_license_key: "{{ lookup('aws_ssm', '/cyhy/core/geoip/license_key', region='us-east-1') }}"
 
 - hosts: cyhy_archive
   name: Install cyhy-archive helper script
   become: yes
   become_method: sudo
   roles:
     - cyhy_archive
+  vars:
+    maxmind_license_key: "{{ lookup('aws_ssm', '/cyhy/core/geoip/license_key', region='us-east-1') }}"
```

Comment:
Why do we need the maxmind key here?

Reply:
Dependency chain. [cisagov/ansible-role-cyhy-archive](https://github.com/cisagov/ansible-role-cyhy-archive) depends on [cisagov/ansible-role-cyhy-core](https://github.com/cisagov/ansible-role-cyhy-core) which depends on [cisagov/ansible-role-geoip2](https://github.com/cisagov/ansible-role-geoip2).

Reply:
Thanks for the reminder! 👍